### ISSUE TITLE
Generation of Observables takes a long time 

### ISSUE THREAD

**Issue by CodeByDrescher** [virtualcell/vcell#1336]
When a rule-based model is saved (or flattened to generate a network), the step of generating observables takes disproportionally long time.

**Comment by danv61** [comment]
Duration before:
```
Adding 8 observables to model, 560 ms
removeObsTime: 300,  addParamTime: 229
```
Duration after the fix:
```
Adding 8 observables to model, 20 ms
removeObs Time: 1,  addParam Time: 0
```

**Comment by danv61** [comment]
fixed in 68f793e
